web/src/components/LoginForm.js

```diff
@@ -0,0 +1,125 @@
+import { Button, Spinner } from '@cmsgov/design-system';
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+import { withRouter } from 'react-router';
```

Comment:
now that there are so many auth routes, we might need to make an actual auth routes file to direct what page should be shown.